src/decode/lzma.rs

```diff
@@ -151,136 +163,76 @@ where
     rep_len_decoder: rangecoder::LenDecoder,
 }
 
-// Initialize decoder with accumulating buffer
-pub fn new_accum<W>(
-    output: lzbuffer::LzAccumBuffer<W>,
-    lc: u32,
-    lp: u32,
-    pb: u32,
-    unpacked_size: Option<u64>,
-) -> DecoderState<W, lzbuffer::LzAccumBuffer<W>>
-where
-    W: io::Write,
-{
-    DecoderState {
-        _phantom: PhantomData,
-        partial_input_buf: std::io::Cursor::new([0; MAX_REQUIRED_INPUT]),
-        output,
-        lc,
-        lp,
-        pb,
-        unpacked_size,
-        literal_probs: vec![vec![0x400; 0x300]; 1 << (lc + lp)],
-        pos_slot_decoder: vec![rangecoder::BitTree::new(6); 4],
-        align_decoder: rangecoder::BitTree::new(4),
-        pos_decoders: [0x400; 115],
-        is_match: [0x400; 192],
-        is_rep: [0x400; 12],
-        is_rep_g0: [0x400; 12],
-        is_rep_g1: [0x400; 12],
-        is_rep_g2: [0x400; 12],
-        is_rep_0long: [0x400; 192],
-        state: 0,
-        rep: [0; 4],
-        len_decoder: rangecoder::LenDecoder::new(),
-        rep_len_decoder: rangecoder::LenDecoder::new(),
+impl DecoderState {
+    // Initialize decoder with accumulating buffer
```

Comment:
There is no accumulating buffer anymore here. IMO the function is self-explanatory, the comment can be removed.

Reply:
https://github.com/gendx/lzma-rs/pull/73/commits/5716a6bca5872c33ff806ccfd6a87b6ea90e31b9#diff-768757151baad4fb0605aaeee5210dc343977125d64711ebcf5a0e755b2f23c0R167